webpages/settings/components/addon-setting.html

```diff
@@ -69,10 +76,18 @@
         type="checkbox"
         class="switch blue"
         v-model="addonSettings[setting.id]"
+        :aria-label="msg('nameSetting', [setting.name])"
```

Comment:
Could the settings be linked to their labels instead?

Reply:
Unless there's some way to append to another element's label, setting it explicitly allows additional information to be included, like saying `"NAME setting, checkbox"`, not just `"NAME, checkbox"`. However, using `aria-labelledby` might be better, for other accessibility reasons.

Alternatively, addon setting blocks could be labeled or marked as regions to make it clear that the contained inputs control setting values. If as a result the entire group of addon settings was announced as `"ADDON_NAME settings group"`, the input label appendage wouldn't be needed.